spec/models/like_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  it 'can be created' do
+    user = User.create(name: 'Test User')
+    post = Post.create(title: 'Test Post')
+
+    like = Like.new(user: user, post: post, text: 'This is a test like.')
+
+    expect(like).to be_valid
```

Comment:
- [ ] Great job for adding the tests for the post validation but I think it would be great if you add tests for the methods here as mentioned in the requirements to test all the methods so please test `increment_post_likes_counter`  that in your like model 🔥⭐♻

![image](https://github.com/Eccb7/Pi-Blogs/assets/58553711/e4c9761f-0e3c-4776-8de3-93d3e888e77f)